app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'prawn'
+require 'prawn/table'
+require 'lighthouse/veterans_health/client'
+
+module FastTrack
+  class DisabilityCompensationJob
+    include Sidekiq::Worker
+
+    extend SentryLogging
+    # NOTE: This is apparently at most about 4.5 hours.
+    # https://github.com/mperham/sidekiq/issues/2168#issuecomment-72079636
+    sidekiq_options retry: 8
+
+    sidekiq_retries_exhausted do |msg, _ex|
+      submission_id = msg['args'].first
+      submission = Form526Submission.new
+      submission.start_evss_submission(_status, submission_id: submission_id)
```

Comment:
I don't think this will work since you are calling `options['submission_id']` in the `start_evss_submission` method. Has this been tested?

Reply:
We haven't tested that particular call, I think because folks weren't sure how to test it. I could look into that on Monday, but @NatashaJIbrahim and @tadhg-ohiggins probably could speak more to the current state of the file.

I updated the `start_evss_submission` method to look for the symbol instead of the string, and updated relevant calls to the method here: https://github.com/department-of-veterans-affairs/vets-api/pull/8558/commits/3bb6eab316154dc4a6801ec42628ab035a829c37

Reply:
Now that this is changed from a hash rocket, you will need to explicitly add brackets otherwise this will be interpreted as a keyword argument.
```suggestion
      submission.start_evss_submission(_status, { submission_id: submission_id })
```